index.spec.js

```diff
@@ -74,6 +74,19 @@ describe('humanFormat()', function () {
       })
     })
 
+    it('should use a simplified custom scale', function () {
+      var scale = new humanFormat.Scale({
+        K: 1000,
+        M: 1000000,
+        B: 1000000000
+      })
+      expect(humanFormat(100, { scale: scale, decimals: 0 })).toBe('100') // fails, this returns "0 K" and there is no way to specify no prefix
```

Comment:
Not a bug, you need to add an empty prefix:

```js
var scale = new humanFormat.Scale({
  '': 1,
  'K': 1e3,
  'M': 1e6,
  'B': 1e9,
})
```

Though I agree it's not very obvious 😛